src/TestContainers.hs

```diff
@@ -25,10 +25,22 @@ module TestContainers
     M.setRm,
     M.setEnv,
     M.setNetwork,
+    M.withNetwork,
```

Comment:
do you intend to expose withNetworkAlias too?

Reply:
Yeah!